### ISSUE TITLE
Annotation support

### ISSUE THREAD

**Issue by artman** [creditkarma/thrift-parser#45]
How do you folks feel about supporting annotations, e.g.:

```
struct foo {
  1: i32 bar ( presence = "required" );
} (
  annotation = "Foo",
  another.annotation = "Bar"
)
```

Currently errors are produced when annotations are encountered.

**Comment by kevin-greene-ck** [comment]
This is definitely on the list of things we want to support in the not too distant future, just haven't had the bandwidth for us to work on it.

**Comment by artman** [comment]
I'd be happy to add support for annotations.

**Comment by kevin-greene-ck** [comment]
Resolved by #46
